.travis.yml

```diff
@@ -0,0 +1,20 @@
+---
+deploy:
+  -
+    provider: script
+    script: "bash scripts/goreleaser.sh"
+    skip_cleanup: true
+    true:
```

Comment:
Could you explain what is the difference between `true` as opposed to the keyword `on`? Just curious here, no need to change this.

Reply:
I think this made it in through copy and paste. I don't see any reason we shouldn't switch it to `on`.